SwiftyAttributes/Sources/iOS_tvOS/UIKit+SwiftyAttributes.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  UIKit+SwiftyAttributes.swift
+//  SwiftyAttributes
+//
+//  Created by Roman Podymov on 26/05/19.
+//  Copyright © 2019 Roman Podymov. All rights reserved.
+//
+
+#if os(macOS)
+#elseif os(watchOS)
+#else
+    import UIKit
+    public typealias NavigationBar = UINavigationBar
+#endif
+
+#if os(macOS)
+#elseif os(watchOS)
+#else
+extension NavigationBar {
+
+    public var swiftyTitleTextAttributes: [Attribute]? {
```

Comment:
could you add some documentation to both of these variables? thanks 🙏

Reply:
> could you add some documentation to both of these variables? thanks 🙏

Added description for these new properties, if you want you can add something better.